src/style.css

```diff
@@ -0,0 +1,36 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
+
+}
+
+.stars--0 .stars__star:nth-child(1),
+.stars--1 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(1),
+.stars--5 .stars__star:nth-child(1) {
+  margin-left: 0;
+}
```

Comment:
It's not okay. Simplify this.

Hint: use only one seletor `.stars__star` + preudo classes: `:firs-child` + `:not`